stix_shifter/stix_translation/src/modules/bigfix/bigfix_to_stix/observable.py

```diff
@@ -0,0 +1,26 @@
+REGEX = {
```

Comment:
This file already exits in json_to_stix so it's best to use that rather than repeat code.

Reply:
yup. i will use that for sure.